nix/default.nix

```diff
@@ -0,0 +1,230 @@
+        "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBHnjecqMe2lrGzAvQ2VQRTXhjZ5q1tONgme+2/97Z3VSXdY0i2bEH3qGEIC7uMyWUfmLystXxqP0u6/Xspmm0Ck="
+      ];
+    in
+    {
+      users.users.root.openssh.authorizedKeys.keys = keys;
+      users.users.komunix = {
+        home = "/home/komunix";
+        createHome = true;
+        isNormalUser = true;
+        extraGroups = [ "wheel" ];
+        openssh.authorizedKeys.keys = keys;
+      };
+    };
+
+  flake.nixosModules.services-cachex =
+    {
+      pkgs,
+      config,
+      lib,
+      ...
+    }:
+    let
+      cfg = config.services.cachex;
+    in
+    with lib;
+    {
+      options.services.cachex = {
+        enable = mkOption {
+          default = false;
+          type = with types; bool;
+          description = ''
+            Enable caddy for cachex
+          '';
+        };
+        enableCron = mkOption {
+          default = false;
+          type = with types; bool;
+          description = ''
+            Enable cachex in cron job
+          '';
+        };
+        cachexPackage = mkOption {
+          type = types.package;
+          description = ''
+            Cachex Package
+          '';
+        };
+        caddyPackage = mkOption {
+          default = pkgs.caddy;
+          type = types.package;
+          description = ''
+            Caddy package
+          '';
+          example = literalExample "pkgs.caddy";
+        };
+        workDir = mkOption {
+          type = types.str;
+          example = literalExample "/home/komunix";
+        };
+        listenAddress = mkOption {
+          type = types.str;
+          default = "127.0.0.1";
+          description = ''
+            Listen Address
+          '';
+        };
+        listenPort = mkOption {
+          type = types.port;
+          default = 2022;
+          description = ''
+            Listen Address
+          '';
+        };
+      };
+      config = mkIf cfg.enable {
+        environment.systemPackages = [ cfg.caddyPackage ];
+
+        services.cron.enable = cfg.enableCron;
+        services.cron.systemCronJobs = [
+          "* * * * * cachex ${getExe cfg.cachexPackage} > ${cfg.workDir}/cachex/index.html"
+        ];
+
+        system.activationScripts.createDir =
+          # bash
+          mkBefore ''
+            [[ -d ${cfg.workDir}/cachex ]] || \
+              (mkdir -p ${cfg.workDir}/cachex && chown komunix:users ${cfg.workDir}/cachex)
+
+            # TODO: better way is using options `services.nfs.*` from `NixOS`.
+            [[ -d ${cfg.workDir}/nfs ]] || \
```

Comment:
Need your help for this @faultables 

https://nixos.wiki/wiki/NFS

Reply:
Yg bagian ini:

![IMG_0426](https://github.com/user-attachments/assets/f1dbbc11-079f-4ddc-b75c-b838be87be3b)

Bisa di cek di /etc/fstab bang iirc. Klo ternyata belum persistent, di history raspi coba cari “mount” nah harusnya sama formatnya hehe (sorry lagi blm bisa ssh)

Reply:
I think ini bener https://github.com/evilfactorylabs/area13/pull/17/files#diff-4e3bd328e4c998f53b19da7639df45b2db63601be4b4343534880adfbeaa5840R206